binr/ragg2/ragg2.c

```diff
@@ -226,6 +228,9 @@ int main(int argc, char **argv) {
 			}
 			}
 			break;
+		case 'S':
+		    str = optarg;
```

Comment:
Use tabs

Reply:
I did use tabs, maybe it's the vscode thing?